app/src/main/res/layout/administrator_controls_app_information_view.xml

```diff
@@ -36,15 +35,14 @@
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
```

Comment:
`android:layout_height="wrap_content"` can be removed as it is already defined in `Caption -> TextView.Common1`